india_compliance/gst_india/utils/test_e_waybill.py

```
@@ -1335,6 +1335,235 @@ def test_generate_e_waybill_with_cancelled_gstin_error_3029_standard(self):
+        def mock_get_value(doctype, name, fieldname, *args, **kwargs):
+            if doctype == "Subcontracting Inward Order Item" and name == "test_scio_detail_123":
+                if fieldname == "parent":
+                    return "SCIO-00001"
+            return original_get_value(doctype, name, fieldname, *args, **kwargs)
+        
+        def mock_get_all(doctype, *args, **kwargs):
+            if doctype == "Subcontracting Inward Order Item":
+                filters = kwargs.get("filters", {})
+                if (filters.get("parent") == "SCIO-00001" and 
+                    filters.get("reference_name") == "test_scio_detail_123" and
+                    filters.get("is_customer_provided_item") == 1):
```

Flake8 (7.3.0)

[error] visually indented line with same indent as next logical line

(E129)